### ISSUE TITLE
Two dividers with primary area in the middle (ultrawide support)

### ISSUE THREAD

**Issue by cognivore** [fuhsjr00/bug.n#259]
For ultrawide and super-ultrawide users to be happier it's crucial to have an option to have two dividers in vertical tiling in such way that the middle area is the primary, and the ones to each sides of it are secondary.

I wonder if that is already implemented somewhere / achievable with configuration or is it a new feature that needs to be implemented.

**Comment by joten** [comment]
Currently, this is not implemented.

There once was a layout used to arrange the subwindows of gimp; but it is not in the current version.

**Comment by cognivore** [comment]
Do we have the code somewhere? I could maybe take a stab on it some time in August.
Currently I'm resorting to PowerToys and while the experience is somewhat smoother in terms of the responsiveness of the window allocation, but the utter lack of hotkeys for _anything_ and lack of independent displays is beyond annoying.

Sorry for a rant, but yeah, I'd like to help some day.

**Comment by joten** [comment]
> Do we have the code somewhere?

Nope, the code is ancient - from around 2008, I think, was in a different repository and maybe is in some sort of a backup. It would be easier to write a new version, since it would have to be integrated in a current version of bug.n anyway.
